### ISSUE TITLE
Create checker for link color contrast stuff

### ISSUE THREAD

**Issue by HRemonen** [HRemonen/SwiftContrast#15]
Links add more complexity to the color contrast requirements.
Links should be underlined, otherwise the text must have at least 3:1 contrast with the surrounding body text. Also links must have hover and focus indicators.

Additionally the link text and the body text must have at least 4.5:1 (small text) contrast with the background or 3:1 with large text.

Create a new "feature" that is used solely for checking the contrast ratios of links

- [ ] Input for link color (the text of the link)
- [ ] Input for the body text color (the color of the text that is surrounding the link)
- [ ] Input for the background color (the color that is surrounding the paragraph or whatever)
